roles/kube-gateway/files/nginx.yaml

```diff
@@ -0,0 +1,10 @@
+init_config:
```

Comment:
Let's actually just build this file into the farva docker container since the file is completely static - this simplifies a lot of things.